### PR TITLE
Add status to bills.

### PR DESCRIPTION
**Pull request by clemenshelm**

### CODE REVIEW

File: client/bills.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <h1>Bills</h1>
   <ul class="bills">
     {{#each bills}}
-    <li class="bill">
+    <li class="bill has-status-{{status}}">
```
Comment (DominikGuzei) [inline client/bills.html]: The status classes might be more beautiful like that: `is-{{status}}` so that it reads: `is-created`

**Comment by DominikGuzei** [comment]
except for the failed build it looks good :+1:

**Comment by clemenshelm** [comment]
Thanks for the feedback. I considered it and also made the test runner smarter.

**Comment by DominikGuzei** [comment]
looks good! :+1:

**Pull request merged**